x/profiles/keeper/genesis_test.go

```diff
@@ -183,14 +186,15 @@ func (suite *KeeperTestSuite) Test_InitGenesis() {
 	ext := suite.GetRandomProfile()
 
 	testCases := []struct {
-		name      string
-		store     func(ctx sdk.Context)
-		genesis   *types.GenesisState
-		shouldErr bool
-		check     func(ctx sdk.Context)
+		name        string
+		setup       func(ctx sdk.Context)
+		store       func(ctx sdk.Context)
```

Comment:
I'm actually quite puzzled on whether we should split `setup` and `store` into two different methods. Considering that in the `setup` we are actually telling how keepers should be answering, I think they can be merged into one single method (leaving the old `store` for backward compatibility and naming conventions). What do you think?

Reply:
From my perspective, `store` should primarily focus on `ctx` statement, while `check` verify `ctx` statement after the operation is executed. As you mentioned, `setup` we are actually telling how keepers should be answering, these keepers are not involved with `ctx` statement, I believe it would be beneficial to separate them into two distinct methods.

Reply:
Yes, removed it.